assets/_scss/home/img_clicker.scss

```diff
@@ -0,0 +1,94 @@
+.no-scroll {
+    overflow: hidden;
+}
+
+#desc {
+    display: none;
+}
+
+.containerTeam img {
+    cursor: pointer
+}
+
+.showPopUp {
+    z-index: 999;
+    display: none;
+    max-height: 80vh;
+    overflow-y: auto;
+}
+
+.showPopUp .overlay {
```

Comment:
Refactor to use `scss`. This element`.overlay` element should be in the previous one `.showPopUp`